doc/changelog/02-specification-language/11132-master+fix-implicit-let-fixpoint-bug3282.rst

```diff
@@ -0,0 +1 @@
+- Bugs fixed in computing implicit arguments for fix/cofix in the presence of let-in, see #3282 (`#11132 <https://github.com/coq/coq/pull/11132>`_, by Hugo Herbelin).
```

Comment:
Does this really justify a changelog entry? It's just an anomaly that disappears IIUC.

Reply:
It's more than an anomaly, it is also some (co)fixpoints with implicit arguments which were wrongly rejected. But I have no good intuition yet on when a changelog is expected and I can remove it if you want.

Reply:
I would rather say that your changelog entries are (similar to your PR title and commit messages) often unclear. In this case, I would have preferred something like "Fixed a bug preventing to define some valid (co)fixpoints with implicit arguments."

Reply:
Ah, sorry. Thanks for telling. Your formulation is nice and I took it as a basis for a reformulation.

For the change logs, will there be a manual pass over them before release to harmonize them and/or removing the one which appear at the end not relevant enough?

Reply:
Yes, this is the role of the RM but something that I help with given the current state that is still somewhat experimental.